components/appointmentPicker/appointmentPickerDataService.js

```diff
@@ -3,8 +3,9 @@ define('appointmentPickerDataService', [], function($http) {
 	function appointmentPickerDataService($http) {
 		return {
 			loadAllAppointments: function(year, appointmentType = "residential") {
+				// Is this able to call anything? I only see getAppointmentTimes in getTimes.php
```

Comment:
Yah, so the `getTimes.php` file is a little special. It only has one action (unlike other files where there's multiple, so you have to specify it). Since there's only one, we don't do the switch statement and just call the method outright (like you noted, it's at `getTimes.php` line 11).

So to answer your question, this line works. But yah the `action=getAppointments` is likely the result of a copy-paste mistake and could be removed.